<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>350</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Nirmala UI Semilight</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>ResiliPy</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(43, 42, 51);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="buttonLabeler">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(66, 65, 77);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Labeler</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="buttonBuilder">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(66, 65, 77);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Builder</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="buttonPreprocessor">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(66, 65, 77);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Preprocessor</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="buttonGithub">
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(66, 65, 77);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>GitHub</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="buttonGuide">
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(66, 65, 77);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Wiki</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
